src/components/TextInput/BaseTextInput/types.ts

```diff
@@ -108,6 +108,12 @@ type CustomBaseTextInputProps = {
     /** Should live markdown be enabled. Changes RNTextInput component to RNMarkdownTextInput */
     isMarkdownEnabled?: boolean;
 
+    /** Clear button should be displayed */
+    shouldShowClearButton?: boolean;
+
+    /** Callback to handle clear the value on Form when input is used in the Form component. */
+    onClearInput?: (value: string | undefined) => void;
```

Comment:
I can't imagine a good reason to add this callback now. I'm not sure if it's better to have it when we have a real use case.

Reply:
I removed ```onClearInput```